RELEASE/scripts/autoscend/paths/g_lover.ash

```diff
@@ -48,33 +46,87 @@ boolean glover_usable(string it)
 	{
 		return true;
 	}
-	item checkItem = it.to_item();
-	if(checkItem != $item[none] && $items[SpinMaster&trade; lathe, // it works since there's no "use" link
+	return false;
+}
+
+boolean glover_usable(effect eff)
+{
+	if(!in_glover())
+	{
+		return true;
+	}
+	if($effects[Stone-Faced] contains eff)
+	{
+		return true;	//explicit exceptions that work in glover despite not having G in the name
+	}
+	return glover_usable(eff.to_string());
+}
+
+boolean glover_usable(skill sk)
+{
+	if(!in_glover())
+	{
+		return true;
+	}
+	
+	// Some of these have g in their name, but are included so it's clearer that all bookshelf skills are allowed.
+	// They only work when cast from the bookshelf itself, unless they have a g in their name already
+	if($skills[summon snowcones, summon stickers, summon sugar sheets, summon clip art, summon rad libs,
+	summon smithsness, summon candy heart, summon party favor, summon love song, summon BRICKOs, summon dice,
+	summon resolutions, summon taffy, summon hilarious objects, summon tasteful items, summon alice\'s army cards,
+	summon geeky gifts, summon confiscated things] contains sk)
+	{
+		return true;
+	}
+
+	return glover_usable(sk.to_string());
+}
+
```

Comment:
Tome skills can still be used, as long as the newly added useTomeSkill and useLibramSkill are called. I didn't add an equivalent useGrimoireSkill because I couldn't find anywhere that grimoire skills were used, although I might have just missed them.